src/directory/directory.js

```diff
@@ -476,57 +486,57 @@ const directory = {
           {
             title: 'Overview',
             route: '/lib/in-app-messaging/overview',
-            filters: ['js']
+            filters: ['react-native']
```

Comment:
it looks like in-app messaging is available for JS and RN but the UI components are only available for RN, is this correct?  do we want to remove JS support here (or did you move it somewhere else)?

Reply:
Without the UI components the category is exceptionally hard to use. For now I'm gonna hide it until we go GA end of september and show it for JS (specifically React).